cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -106,10 +107,22 @@ func NewCMServer() *CMServer {
 		PodEvictionTimeout:                5 * time.Minute,
 		ClusterName:                       "kubernetes",
 		EnableHorizontalPodAutoscaler:     false,
+		VolumeConfigFlags: VolumeConfigFlags{
+			// default values here
+			PersistentVolumeRecyclerTimeoutNFS: 300,
+		},
 	}
 	return &s
 }
 
+// VolumeConfigFlags is used to bind CLI flags to variables.  This top-level struct contains *all* enumerated
+// CLI flags meant to configure all volume plugins.  From this config, the binary will create many instances
+// of volume.VolumeConfig which are then passed to the appropriate plugin. The ControllerManager binary is the only
+// part of the code which knows what plugins are supported and which CLI flags correspond to each plugin.
+type VolumeConfigFlags struct {
+	PersistentVolumeRecyclerTimeoutNFS int
```

Comment:
Each thing we want to override would need to be explicitly defined in the binary (i.g, this struct).

Because timeout's can differ by plugin, each gets its own attribute in `VolumeConfigFlags`.

The generic re-use comes from mapping VolumeConfigFlags to VolumeConfig.